charter.md

```diff
@@ -32,32 +28,29 @@ A clear scope is particularly important where patent licensing obligations may a
 
 ## Deliverables
 
-{TBD: Provide a brief description of each specification the group plans to produce.
-Where an estimate is possible, it can be useful to provide an estimated schedule for key deliverables.
-As described below, the group may later modify the charter deliverables. }
-
 ### Community Group Reports that are Specifications
 
-{TBD: Provide a brief description of each Community Group Report the group plans to publish.
-The description should be enough for those considering participation to know
-the general types of technologies that will be defined in the report.}
-
-### Community Group Reports that are not Specifications {TBD: Include only as necessary}
```

Comment:
I'm planning to add this and the software section back in to cover:
* A security model and pairing ceremony
* Polyfills for the Web Bluetooth API on top of various native APIs, including [Chrome Apps](https://github.com/WebBluetoothCG/chrome-app-polyfill) and Cordova.